Horion/Module/Modules/AutoJump.cpp

```diff
@@ -0,0 +1,17 @@
+#include "AutoJump.h"
+
+AutoJump::AutoJump() : IModule(0, Category::MOVEMENT, "Automatically jump") {
+}
+
+AutoJump::~AutoJump() {}
+
+const char* AutoJump::getModuleName() {
+	return ("AutoJump");
+}
+
+void AutoJump::onTick(C_GameMode* gm) {
+	auto player = g_Data.getLocalPlayer();
+	if (player == nullptr) return;
```

Comment:
i dont think this is needed because it's ontick

Reply:
what

Reply:
visual studio gives me errors without it.

Reply:
i mean the if (player == nullptr) return;